charts/kubetail/values.yaml

```diff
@@ -95,6 +118,7 @@ ingress:
 
 # kubetail config (see https://github.com/kubetail-org/kubetail for defaults)
 config:
+  # namespace: '{{ .Release.Namespace }}'
```

Comment:
Let's remove this comment so as not to confuse users looking at values.yaml

Reply:
Done